Feature/1054 statistics hover tooltip

allthegoldt opened the pull request:
clarat-org/clarat#1054
Statistics: Hover in BurnUp Chart
clarat-org/clarat#1055
Moved legacy code from statistics and sorted the rest of the burnUpChart files.

Commit:
d3 test

Commit:
clarat-org/clarat#1054 Statistics Hover in BurnUp Chart

clarat-org/clarat#1055

Refactored some of the statistics code and made a hover statistics
display

Commit:
clarat-org/clarat#1054 Burnup Chart Hover Statistics

The chart shows newest data (data, that days’ points, cumulated points)
per default and changes data on mouseover.

Commit:
clarat-org/clarat#1055 Moved legacy code

Moved legacy code from statistics

Commit:
Merge branch 'develop' into feature/1054_statistics_hover_tooltip

KonstantinKo reviewed (changes requested):
(nach mündlicher Absprache)

Commit:
clarat-org/clarat#1054

clarat-org/clarat#1055
Moved legacy code in the legacy code folder and removed from Routes
clarat-org/clarat#1054
Moved most variables to the container, using only one dispatch, added
coordinate-lines that are fixed on the data-line, and chart data only
showed on mouseover

Commit:
Merge branch 'develop' into feature/1054_statistics_hover_tooltip

# Conflicts:
#	client/app/bundles/Dashboard/components/PersonalStatisticCharts.jsx

Commit:
clarat-org/clarat#1054

clarat-org/clarat#1055
Merged with develop, changed filenames to divide in personal/team not
sure about naming but suppose those could change anyway.

KonstantinKo committed:
for clarat-org/clarat#1054 - small indentation, comment, line length changes

KonstantinKo committed:
for clarat-org/clarat#1054 - change hover event to click, format date

Commit:
Merge branch 'develop' into feature/1054_statistics_hover_tooltip

# Conflicts:
#	Gemfile
#	Gemfile.lock
#	db/schema.rb

Commit:
Mergeconflict

Commit:
Removed surplus files

clarat-org/clarat#1054
